docs/application/dotnet/samples/system/overview.md

```diff
@@ -34,28 +49,35 @@
 			<td><img alt="" height="267" src="media/m23sensor.png" width="150"/></td>
 			<td>
 			<p><strong>(M) Sensor</strong> [In progress]</p>
-			<p>This sample application demonstrates how you can managing sensors and receiving sensor data from the sensor device.</p>
+			<p>This sample application demonstrates how to manage sensors and receive sensor data from the sensor device.</p>
 			</td>
 		</tr>
 		<tr>
 			<td><img alt="" height="267" src="media/m4systeminfo.png" width="150"/></td>
 			<td>
 			<p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Mobile/UI/System-info" target="_blank"><strong>(M) System-info</strong></a></p>
-			<p>This sample application demonstrates how to get information about system properties and capabilities of the device.</p>
+			<p>This sample application demonstrates how to get information about the system properties and capabilities of the device.</p>
 			</td>
 		</tr>
 		<tr>
 			<td><img alt="" height="267" src="media/m46systeminfo.png" width="150"/></td>
 			<td>
 			<p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Mobile/SystemInfo" target="_blank"><strong>(M) SystemInfo</strong></a></p>
-			<p>This sample application demonstrates how to obtain data provided by the system using <a href="https://developer.tizen.org/dev-guide/csapi/api/Tizen.System.html" target="_blank">Tizen.System C# API</a>.</p>
+			<p>This sample application demonstrates how to obtain the data provided by the system using <a href="https://developer.tizen.org/dev-guide/csapi/api/Tizen.System.html" target="_blank">Tizen.System C# API</a>.</p>
 			</td>
 		</tr>
 		<tr>
 			<td><img alt="" height="180" src="media/wsystemInfo.png" width="180"/></td>
 			<td>
 			<p><strong>(W) SystemInfo</strong> [In progress]</p>
-			<p>This sample application demonstrates how to obtain data provided by the system using <a href="https://developer.tizen.org/dev-guide/csapi/api/Tizen.System.html" target="_blank">Tizen.System C# API</a></p>
+			<p>This sample application demonstrates how to obtain data provided by the system using <a href="https://developer.tizen.org/dev-guide/csapi/api/Tizen.System.html" target="_blank">Tizen.System C# API</a> and how to create Circular UI using <a href="https://github.com/Samsung/Tizen.CircularUI/" target="_blank">Tizen.Wearable.CircularUI C# API</a>.</p>
+			</td>
+		</tr>
+		<tr>
+			<td><img alt="" height="180" src="media/wsystem_info.png" width="180"/></td>
+			<td>
```

Comment:
Why there is a duplication?
Why <p><strong>(W) SystemInfo</strong> [In progress]</p> section is written twice with same description?
Please verify and update.